test/gateway/ost_prime/increase_supply.js

```diff
@@ -0,0 +1,206 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const OSTPrime = artifacts.require("TestOSTPrime")
+  , BN = require('bn.js');
+
+const Utils = require('../../../test/test_lib/utils');
+const EventDecoder = require('../../test_lib/event_decoder.js');
+const NullAddress = "0x0000000000000000000000000000000000000000";
+
+contract('OSTPrime.increaseSupply()', function (accounts) {
+
+  const DECIMAL = new BN(10);
+  const POW = new BN(18);
+  const DECIMAL_FACTOR = DECIMAL.pow(POW);
+  const TOKENS_MAX = new BN(800000000).mul(DECIMAL_FACTOR);
+
+  let brandedTokenAddress,
```

Comment:
It seems your IDE indentation is 2 instead of 4 🤔

Reply:
All js files should have indentation as 2. We are following this for all repository.